tests/django/test_command.py

```diff
@@ -142,11 +142,12 @@ def test_call_command(capsys):
       unique: false
     char_choices:
       generator: Choices
-      choices: [foo, bar]
+      choices:
+      - foo
+      - bar
     int_choices:
       generator: Choices
-      choices: [1, 2]
-""".format(
-        username_length=30 if django.VERSION < (1, 10) else 150,
-        lastname_length=30 if django.VERSION < (2,) else 150,
-    )
+      choices:
+      - 1
+      - 2
+"""
```

Comment:
Files should always end with a new line

Reply:
I'm still on the process of migration from pycharm to vscode. I've upgraded my configs, thanks.